tests/components/screenlogic/test_config_flow.py

```diff
@@ -213,9 +263,12 @@ async def test_option_flow(hass):
 
 async def test_option_flow_defaults(hass):
     """Test config flow options."""
-    entry = MockConfigEntry(domain=DOMAIN, data={}, options=None)
+    entry = MockConfigEntry(domain=DOMAIN)
     entry.add_to_hass(hass)
 
+    await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
Patch the integration setup functions or at least the library to avoid I/O.

Reply:
Commit https://github.com/home-assistant/core/pull/48136/commits/df0a10b963e7a75fe5cfbf2df1d25f68948de543